Docs/Configuration.tex

```diff
@@ -2998,66 +3018,68 @@ \subsection{Boot Properties}\label{miscbootprops}
     \item Entry is system (e.g. \texttt{Reset NVRAM}).
   \end{itemize}
 
-  To see all entries picker menu needs to be reloaded in extended mode by pressing
-  \texttt{Spacebar} key. Hiding auxiliary entries may increase boot performance
-  for multidisk systems.
-
+  To display all entries, the picker menu can be reloaded into ``Extended Mode'' by pressing the
```

Comment:
Hmm, I see where it comes from… I agree that we should pick one, but maybe `Advanced` is really better.

Reply:
Maintained "Extended"